### ISSUE TITLE
editable bool field

### ISSUE THREAD

**Issue by schel4ok** [z-song/laravel-admin#4475]
Laravel Version: 7.2.0
PHP Version: 7.2.19
Laravel-admin: 1.7.14

### Description:
How to make boolean field editable?
When click value changes from 0 to 1 and vice versa.

### Something like that
```
        $grid->column('is_published', 'Опубликовано')->bool()->editable();       
```

**Comment by mRamadan0101** [comment]
you must add this column in your form

**Comment by schel4ok** [comment]
I already opened similar issue about wrong behaviour of select field here https://github.com/z-song/laravel-admin/issues/4251

**Comment by schel4ok** [comment]
can somebody help on this issue?
Obviously select field doesn't work as it is explained in documentation.

I found working solution, but it works only with eloquent relationships.
Working is 
```
$form->select('category_id')->options(NewsPost::selectOptions());
```
instead of this ([explained in documentation](https://laravel-admin.org/docs/#/en/model-form-fields?id=multiple-select)) 
```
 $form->select('category_id')->options(NewsPost::all()->pluck('category_id'));
```
As result you will see Parent category in select dropdown field.

I found that looking in the code of admin menu controller.
But it doesn't work for another fields. 
So this will not work  `$form->select('is_published')->options(NewsPost::selectOptions());`
